app/src/main/java/com/example/keniphhomemaintenance/UtilitiesScreen.kt

```diff
@@ -0,0 +1,16 @@
+package com.example.keniphhomemaintenance
+
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.stringResource
+
+@Composable
+fun UtilitiesScreen(name: String?, address: String?) {
+    Box(contentAlignment = Alignment.Center, modifier = Modifier.fillMaxSize()) {
+        Text(text = stringResource(R.string.utilities_screen_placeholder, name as Any, address as Any))
```

Comment:
Does this require "as Any" for the placeholders? Hint: it does not, the compiler warning is a little misleading

Reply:
I could just replace these with `as String`, but I wrapped them in ?.let calls and then did as String